tests/langservice/test_translation_cache.py

```diff
@@ -0,0 +1,395 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import tempfile
+import json
+import os
+from pathlib import Path
+from unittest.mock import patch, MagicMock
+
+from garak.langproviders.base import LangProvider, TranslationCache
+from garak.langproviders.local import Passthru
```

Comment:
The `PassThru` class should not utilize a cache, please refactor these tests to unit test other `langproviders`, mocking the `_translate()` function for any other class extending `garak.langproviders.base` should be viable.

This applies to both test files added in the PR.